Library/Formula/pxz.rb

```diff
@@ -12,31 +12,83 @@ class Pxz < Formula
     sha256 "aa8d6ad7fb7e1ee38e26e97cd9fcbc23dcf40cc44dea5cece306bf0556322c1a" => :mavericks
   end
 
+  head do
+    url "https://github.com/jnovy/pxz.git"
+
+    # Rebased version of an upstream PR to fix the build on OS X
+    # https://github.com/jnovy/pxz/pull/5
+    patch :DATA
+  end
+
+  depends_on "gcc" => :run
```

Comment:
This feels a lot like a hack. The “proper” way to declare an OpenMP dependency is to say `needs :openmp` (like `needs :cxx11`) instead of the `fails_with :clang`. Unfortunately, that's not going to pull in the compiler as a dependency. I guess nobody noticed until now that OpenMP also requires runtime support, so it is a bad idea to remove the formula that also contains that. We might have to create an `OpenMPRequirement` class activated via `depends_on :openmp` to deal with that properly.